app/controllers/catalog_controller.rb

```diff
@@ -6,6 +6,29 @@ class CatalogController < ApplicationController
   include DateFacetConfigurations
 
   before_action :limit_facets_on_home_page, only: [:index]
+  before_action :adjust_lazy_limits, only: [:index]
+
+  # The subset of facets that are displayed on the home page.
+  # (Before a user clicks "Show more facets")
+  HOME_FACETS = [
```

Comment:
if the only other slow ones are the wps and its brethren, maybe Andrew would want them to display on the home page?